### ISSUE TITLE
[no-duplicates] prefer-inline should merge type and non-type imports

### ISSUE THREAD

**Issue by latin-1** [import-js/eslint-plugin-import#2675]
Input:

```ts
import { A } from "./mod";
import type { B } from "./mod";
```

Output:

```ts
import { A, type B } from "./mod";
```

**Comment by latin-1** [comment]
There is also a regression in 2.27:

Input:

```ts
import { A } from "./mod";
import { type B } from "./mod";
```

Output:

```ts
// 2.26
import { A, type B } from "./mod";
```

```ts
// 2.27
import { A } from "./mod";
import { type B } from "./mod";
```

**Comment by latin-1** [comment]
cc @snewcomer

**Comment by simmo** [comment]
I might be mistaken, but I think the docs might be wrong. As I understand it, it is supposed to be `["error", "prefer-inline"]` to align with `import/consistent-type-specifier-style`, but the only allowed syntax is `["error", {"prefer-inline": true}]`. I'm easy either way just thought I'd flag it 😄

**Comment by ljharb** [comment]
@simmo that doesn't seem right - the schema is clear that it's a string, not an object. Are you sure you're using the latest version of the plugin?

**Comment by simmo** [comment]
Yep I'm using `2.27.5`.  It allows object syntax but not string.
<img width="646" alt="image" src="https://user-images.githubusercontent.com/265345/212983876-6206936c-8298-4cbb-b967-917ef9b96e11.png">

**Comment by ljharb** [comment]
ohhh yes, that's correct, they have a different schema (i was checking consistent-type-specifier-style).

I don't think it's worth adding a string overload for no-duplicates, nor an object overload for consistent-type-specifier-style, but I agree it ends up being an inconsistency. If the docs are wrong, a PR to fix them would be appreciated!

**Comment by simmo** [comment]
> If the docs are wrong, a PR to fix them would be appreciated!

My pleasure 😄 - PR for documentation #2684
